tests/integration/api/test_get_collections.py

```diff
@@ -4,13 +4,13 @@
 from .utils import assert_response_status
 
 @pytest.mark.parametrize("endpoint, expected_status", [
-    ("/collection/3650664c-c8be-4d07-8d64-2d7003b02048", 200),
-    ("/collection/00000000-0000-0000-0000-000000000000", 404),
-    ("/collection/invalid_id_format", 400),
-    ("/collection/", 404),
-    ("/collection/%$@!*", 400)
+    ("/collections/3650664c-c8be-4d07-8d64-2d7003b02048", 200),
+    ("/collections/00000000-0000-0000-0000-000000000000", 404),
+    ("/collections/invalid_id_format", 400),
+    ("/collections/", 404),
+    ("/collections/%$@!*", 400)
 ])
-def test_get_collection(endpoint, expected_status):
+def test_get_collections(endpoint, expected_status):
```

Comment:
I think since we are just getting a single collection with an id, this can be test_get_collection